src/google_apps_clj/google_calendar.clj

```diff
@@ -11,9 +11,10 @@
            (com.google.api.services.calendar Calendar
                                              Calendar$Builder
                                              Calendar$Events$Insert
-                                             CalendarScopes)))
+                                             CalendarScopes)
+           (com.google.api.client.googleapis.auth.oauth2 GoogleCredential)))
 
-(t/ann build-calendar-service [cred/GoogleCtx -> Calendar])
+(t/ann build-calendar-service [(t/U cred/GoogleCtx GoogleCredential) -> Calendar])
```

Comment:
Given the repeated use of this union type throughout, I'd think we might want to defalias it in the creds ns. Maybe... `Authentication` or just plain ol' `Credential`?